app/AppKernel.php

```diff
@@ -32,6 +32,12 @@
 
 class AppKernel extends Kernel
 {
+    const VERSION = '1.7.4.0';
+    const MAJOR_VERSION_STRING = '1.7';
```

Comment:
Not sure it's a good idea to register in const `MAJOR`, `MINOR` and `RELEASE` const, we can get this value from `VERSION`. We can retrieve this information from the string. Making this more generic can be useful for the community in case they also need to manage version (from dependencies for example). :)

Reply:
@PierreRambaud I did it because Symfony did it :rofl:  (https://github.com/symfony/http-kernel/blob/master/Kernel.php) I don't know why they did, I just thought there must be some good reason. Should I remove it still? - What do you mean by "manage version from dependencies"?

Reply:
In this case, we can still have it in dependency injection.
If you have modules or themes for example, which need to manage some versions for anything like API versions, dependencies versions, modules versions or manage his own version, having a class to do it will be so useful for everyone. No need to rebuild the wheel :)